content/kotsadm/installing/airgap-packages.md

```diff
@@ -8,6 +8,34 @@ draft: false
 
 When installing an application from an airgap package, the container images and application manifests are provided by the application vendor in an archive that can be used to deliver the artifacts into the cluster. This feature is only available for licenses that have the airgap feature enabled.
 
+## Kots install
+Begin by [installing the Kots CLI kubectl plugin](/kots-cli/getting-started/).  The Admin Console can be installed using the CLI and the airgap package included in the [release assets](https://github.com/replicatedhq/kots/releases) named `kotsadm.tar.gz`.   The asset version has to match the CLI version, which can be determined by running:
+
+```shell
+kubectl kots version
+```
+
+First step is to extract kotsadm images and push them into a private registry.  Registry credentials provided in this step need to have push access.  These credentials will not be stored anywhere.
+
+```shell
+kubectl kots admin-console push-images ./kotsadm.tar.gz private.registry.host/kotsadm --registry-username rw-user --registry-password rw-password
+```
+
+The next step is to install the Admin Console using images pushed in the previous step.  Registry credentials provided in this step only need to have read access, and they will be stored in a kubernetes secret in the same namespace where Admin Console will be installed.
+
+```shell
+kubectl kots install --kotsadm-registry private.registry.host/kotsadm --registry-username ro-user --registry-password ro-password application-name
+```
+
+Once this has completed, the kots plugin will create a port-forward to the Admin Console interface. The Admin Console API and Web server are exposed over a ClusterIP service in the namespace provided. The port-forward will be active as long as the CLI is running. Pressing Ctrl+C will end the port forward.
+
+```shell
+  • Press Ctrl+C to exit
+  • Go to http://localhost:8800 to access the Admin Console
+```
+
+Once this message is printed in the terminal, click the link, or visit `http://localhost:8800` to complete the setup using the Admin Console web-based UI.
```

Comment:
Once this message is displayed visit `http://localhost:8800` to complete the application setup using the Admin Console.